### ISSUE TITLE
Wizard framework

### ISSUE THREAD

**Issue by tpietzsch** [TrNdy/mastodon-tracking#3]
Random notes (to be clarified):
* WizardPanelDescriptor.getNextPanelDescriptorIdentifier()
    Should this be hard-wired?
    It would be good if panels could be reused in different sequences
(see #11).

* Support Retina resolution in TransitionAnimator/ImageHelper

* WizardLogService: Should this be a LogService or just capture logs from the standard LogService --> talk to @maarzt

* WizardPanel buttons: Next -> Finish?

Also: Sometimes it is not intuitive that clicking "next" will trigger a costly computation step.
For example, after setting detection parameters, "next" triggers computation on the whole dataset. It would be good if we could draw attention to this. Like "next" goes to a panel with a "compute" button. So that if you "accidentally" clicked "next" you still have a chance to go back before starting the computation. Of course, you should be able to click "next" until the computation is done. Or changing text in the next button to something that alerts you to the fact that "next" is the computation. I don't know. But it seems that there is room for improvement...

**Comment by tinevez** [comment]
Addressed elsewhere or deferred.
